### ISSUE TITLE
Adding or Editing a Category Goal

### ISSUE THREAD

**Issue by bpaulien** [justbudget/justbudget#71]
When I select a category and edit it, with the intention of adding/changing a goal, I tab into the Category Goal field of the Add Budget Category, and the cursor goes to the end of whatever value is there (default 0.00).  
![image](https://user-images.githubusercontent.com/6487363/176664381-31101c69-fadf-4958-99d4-b6c90e51302f.png)
If I don't notice this, and delete it, I end up with something like this:
![image](https://user-images.githubusercontent.com/6487363/176664529-0fa1e55e-cc1f-48f5-bb90-470f9d24fad1.png)
I think it would be better to have the value of the category goal selected by default, so as you start typing, it replaces what is there.

**Comment by justbudget** [comment]
> I think it would be better to have the value of the category goal selected by default, so as you start typing, it replaces what is there.

I agree with this solution, I kind of feel like I should start putting this behavior into more places in the app. When you move to a new text field, it should auto-select the content by default. 

Thanks for the suggestion!
